Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -367,6 +367,7 @@ - (void)deleteTokenWithAuthorizedEntity:(NSString *)authorizedEntity
   if (!handler) {
     FIRInstanceIDLoggerError(kFIRInstanceIDMessageCodeInstanceID001,
                              kFIRInstanceIDInvalidNilHandlerError);
+    return;
```

Comment:
Do we still want to delete the token if there's no completion handler, or leave it as is?

Reply:
the API interface says it is nonnull here so it is supposed to return if passing anil.